src/components/Explorer.js

```diff
@@ -82,18 +79,12 @@ class Explorer extends React.Component {
     if (!check(filters.timeFrom)) filters.timeFrom = appState.filters.timeFrom
     if (!check(filters.timeTo)) filters.timeTo = appState.filters.timeTo
 
-    dispatch(updateFilters(filters))
```

Comment:
this bit (85-90) duplicates a lot of stuff from original `updateFiltersAndUrl` (sans the url updating)